tests/configurable_cluster.py

```diff
@@ -99,6 +107,7 @@ def tear_down(self):
             service presto stop
```

Comment:
This set of commands gets run before every test so deleting the rpm_cache_dir defeats the purpose of that directory. I also can't think of a global setUp()/teardown() method that could delete this directory. As for mount_dir, yes that gets deleted, see the last line of this command.